extensions/amp-base-carousel/amp-base-carousel.md

```diff
@@ -0,0 +1,262 @@
+  text: Displays multiple similar pieces of content along a horizontal axis or vertical axis.
+---
+<!---
+Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# amp-base-carousel
+
+A generic carousel for displaying multiple similar pieces of content along a horizontal or vertical axis.
+
+<table>
+  <tr>
+    <td width="40%"><strong>Availability</strong></td>
+    <td><div><a href="https://www.ampproject.org/docs/reference/experimental.html">Experimental</a>; You must turn on the `amp-base-carousel` experiment to use this component.</div></td>
+  </tr>
+  <tr>
+    <td width="40%"><strong>Required Script</strong></td>
+    <td><code>&lt;script async custom-element="amp-base-carousel" src="https://cdn.ampproject.org/v0/amp-base-carousel-0.1.js">&lt;/script></code></td>
+  </tr>
+  <tr>
+    <td class="col-fourty"><strong><a href="https://www.ampproject.org/docs/guides/responsive/control_layout.html">Supported Layouts</a></strong></td>
+    <td>
+      fill, fixed, fixed-height, flex-item, intrinsic, nodisplay, and responsive.
+    </td>
+  </tr>
+  <tr>
+    <td width="40%"><strong>Examples</strong></td>
+    <td>
+      <a href="https://ampbyexample.com/components/amp-base-carousel/">amp-carousel example</a>
+    </td>
+  </tr>
+</table>
+
+## Behavior
+
+Each of the `amp-base-carousel` component’s immediate children is considered an item in the carousel. Each of these nodes may also have arbitrary HTML children.
+
+The carousel consists of an arbitrary number of items, as well as optional navigational arrows to go forward or backwards a single item.
+
+The carousel advances between items if the user swipes or uses the customizable arrow buttons.
+
+## Attributes
+### Using Media Queries
+
+The attributes for `<amp-base-carousel>` can be configured to use different options based on a media query. You can also use a value without any media queries. The format looks like:
+
+```html
+<amp-base-carousel attr-name="(min-width: 1000px) valueOne, (min-width: 600px) valueTwo, defaultValue"></amp-base-carousel>
+```
+
+The media queries are evaluated from left to right, with the first matching media query being used. A default value (without a media query) is required. In this case, if the page has a screen width of 1000px or more, `valueOne` is used. If the width is between 999px and 600px, `valueTwo` is used. When it is 599px or smaller, `defaultValue` is used.
+
+### Configuration Options
+
+<table>
```

Comment:
I would make each of these configurations their own table.

Reply:
Done.